omnigan/trainer.py

```diff
@@ -255,7 +180,7 @@ def setup(self):
         if self.opts.train.resume:
             self.resume()
 
-        self.set_losses()
+        self.losses = set_losses(opts=self.opts, verbose=self.verbose)
```

Comment:
your function doesn't have named parameters, only positional ones so it should be `(self.opts, self.verbose)` or change the function's signature in `losses.py`